ariadne/executable_schema.py

```diff
@@ -37,8 +54,103 @@ def make_executable_schema(
 
     assert_valid_schema(schema)
 
+    for type_name, field_name, arg, _ in find_enum_values_in_schema(schema):
```

Comment:
This code block should live in separate `validate_schema_enum_values` function

Reply:
Done.